documentation/src/main/asciidoc/chapter-10-advanced-mapping-options.asciidoc

```diff
@@ -60,7 +60,7 @@ public interface SourceTargetMapper {
 ----
 ====
 
-The example demonstrates how the source properties `time` and `format` are composed into one target property `TimeAndFormat`. Please note that the fully qualified package name is specified because MapStruct does not take care of the import of the `TimeAndFormat` class (unless it's used otherwise explicitly in the `SourceTargetMapper`). This can be resolved by defining `imports` on the `@Mapper` annotation.
+The example demonstrates how the source properties `time` and `format` are composed into one target property `TimeAndFormat`. Please note that the fully qualified package name is specified because MapStruct does not take care of the import of the `TimeAndFormat` class (unless it's used otherwise explicitly in the `ErroneousSourceTargetMapperWithIgnoreByDefault`). This can be resolved by defining `imports` on the `@Mapper` annotation.
```

Comment:
Attention, I think you changed it here unintentionally (due to an IntelliJ rename)